webpack.prod.js

```diff
@@ -4,14 +4,15 @@ const { merge } = require('webpack-merge');
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
 const TerserPlugin = require('terser-webpack-plugin');
 const commons = require('./webpack.commons.js');
+const BASE_HREF = process.env.BASE_HREF || '/';
 
 module.exports = merge(commons, {
   mode: 'production',
   devtool: 'source-map',
   output: {
     filename: '[name].[hash].min.js',
     path: path.resolve(__dirname, 'dist'),
-    publicPath: '/calendar/'
+    publicPath: BASE_HREF
```

Comment:
this one is link to https://github.com/OpenPaaS-Suite/esn-frontend-common-libs/issues/193